src/mscorlib/src/System/Globalization/CultureData.cs

```diff
@@ -159,18 +159,19 @@ internal class CultureData
         private String sCompareInfo; // Compare info name (including sorting key) to use if custom
         private String sScripts; // Typical Scripts for this locale (latn;cyrl; etc)
 
-        // CoreCLR depends on this even though its not exposed publicly.
-#if !FEATURE_CORECLR
+        private String sISO639Language2; // 3 char ISO 639 lang name 2 ex: eng
+        private String sAbbrevLang; // abbreviated language name (Windows Language Name) ex: ENU
         private int iDefaultAnsiCodePage = undef; // default ansi code page ID (ACP)
         private int iDefaultOemCodePage = undef; // default oem code page ID (OCP or OEM)
+
+        // CoreCLR depends on this even though its not exposed publicly.
+#if !FEATURE_CORECLR
```

Comment:
Can you please take this a step further and diff the S.P.CoreLib with the reference mscorlib we are trying to match (I can send you a pointer to that offline) and see what else is left out in the globalization namespace? I really want to get this full set exposed.